com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.krioltech.fnmap
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=xdg-documents
```

Comment:
why do you need home access for `nmap`? Also these two filesystem permission are redundant.

Reply:
I need it for saving and loading nmap scan xml files.

Reply:
Redundant home permission removed.